packages/core/admin/admin/src/pages/Settings/pages/ApiTokens/EditView/EditViewPage.tsx

```diff
@@ -190,7 +190,8 @@ export const EditView = () => {
         const res = await createToken({
           ...body,
           // lifespan must be "null" for unlimited (0 would mean instantly expired and isn't accepted)
-          lifespan: body?.lifespan || null,
+          lifespan:
```

Comment:
Just to confirm, the original PR updated both API and Transfer tokens to be more similar (I think 6 file changes in total). Did you check if we lost any of the other changes in the merge that might need to be copied over too?

Reply:
Nope we didn't lose any other changes, the server changes weren't lost